### PR TITLE
add help info

### LINKED ISSUE #209: Package readme
As a [role], I want to [action], so that I can [goal]

## Definition of Done
- [ ] Criterion 1
- [ ] Criterion 2

## Tasks
- [ ] Task 1
- [ ] Task 2

### PR DESCRIPTION
**Pull request by ConnorJacksonBSU**
closes #209

### CODE REVIEW

File: src/views/PopupView.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import * as React from "react";
 import * as cameraControlsImg from "../../assets/images/camera-controls.png";
 import * as selectionControlsImg from "../../assets/images/selection-controls.png";
-import * as timelineControlsImg from "../../assets/images/timeline-controls.png";
```
Comment (PTLux1517) [inline src/views/PopupView.tsx:4]: With Javi's PR being recently merged, it looks like this was accidentally deleted. This should be added back in

File: src/views/PopupView.tsx [hunk 2]
```diff
@@ -34,17 +33,74 @@ export default function PopupView(
                 <img src={selectionControlsImg} alt={"selection controls help image"}
                      style={controlsHelpImgNum===1 ? {display: "block"} : {display: "none"}}
                 />
-                <img src={timelineControlsImg} alt={"timeline controls help image"}
-                     style={controlsHelpImgNum===2 ? {display: "block"} : {display: "none"}}
-                />
```
Comment (PTLux1517) [inline src/views/PopupView.tsx:39]: Same with these 3 lines

File: src/views/PopupView.tsx [hunk 3]
```diff
@@ -34,17 +33,74 @@ export default function PopupView(
...
+                <div style={controlsHelpImgNum===2 ? {display: "block"} : {display: "none"}}>
```
Comment (PTLux1517) [inline src/views/PopupView.tsx:36]: The image number for this will have to be 3 now

File: src/views/PopupView.tsx [hunk 4]
```diff
@@ -34,17 +33,74 @@ export default function PopupView(
...
+                    </ul>
+                </div>
             </div>
             <div id={"menu-dropdown-container"}>
                 <div id={"menu-title"}>Main Menu</div>
                 <dl id={"menu-options"}>
                     <dt>Controls Help</dt>
                     <dd onMouseOver={()=>setControlsHelpImgNum(0)}>camera</dd>
                     <dd onMouseOver={()=>setControlsHelpImgNum(1)}>selection</dd>
-                    <dd onMouseOver={()=>setControlsHelpImgNum(2)}>timeline</dd>
```
Comment (PTLux1517) [inline src/views/PopupView.tsx:47]: This needs to be added back in

File: src/views/PopupView.tsx [hunk 5]
```diff
@@ -34,17 +33,74 @@ export default function PopupView(
...
+                    <dd onMouseOver={()=>setControlsHelpImgNum(2)}>more</dd>
```
Comment (PTLux1517) [inline src/views/PopupView.tsx:103]: The image number for this will have to be 3 now too

Comment (ConnorJacksonBSU) [reply src/views/PopupView.tsx]: i believe i resolved the issues

Comment (PTLux1517) [reply src/views/PopupView.tsx]: Looks good. The only other thing that might be better would be to put the "more" option below the "timeline" option, i.e. swap lines 106 and 107.

Comment (ConnorJacksonBSU) [reply src/views/PopupView.tsx]: k i did it

**Review by javitrejo14 — APPROVED** [review]

**Pull request merged**